spec/rspec/core/formatters/snippet_extractor_spec.rb

```diff
@@ -250,6 +250,24 @@ def another_expression(*)
         end
       end
 
+      context 'when Ripper cannot parse the source', :isolated_directory do
+        let(:file_path) do
+          'invalid_source.rb'
+        end
+
+        let(:line_number) do
+          1
+        end
```

Comment:
Not a merge blocker, but I think I'd prefer the `let` declarations were on one line:

``` ruby
let(:file_path) { "invalid_source.rb" }
let(:line_number) { 1 }
```

Why take up 7 lines when it can be 2?